frontend/src/uploads/UploadContainer.js

```diff
@@ -127,41 +128,52 @@ const UploadContainer = () => {
   return (
     <div className="row">
       <div className="col-12 mr-2">
-        <AlertDialog
-          open={openDialog}
-          title={'Replace existing data?'}
-          dialogue={'Selecting replace will delete all previously uploaded records for this dataset'}
-          cancelText={'Cancel'}
-          handleCancel={handleReplaceDataCancel}
-          confirmText={'Replace existing data'}
-          handleConfirm={handleReplaceDataConfirm}
-        />
-        <Stack direction="column" spacing={2}>
-          <Paper square variant="outlined">
-            <UploadPage
-              alertElement={alertElement}
-              uploadFiles={uploadFiles}
-              datasetList={datasetList}
-              doUpload={doUpload}
-              setDatasetSelected={setDatasetSelected}
-              datasetSelected={datasetSelected}
-              setUploadFiles={setUploadFiles}
-              setReplaceData={setReplaceData}
-              replaceData={replaceData}
-              handleRadioChange={handleRadioChange}
-              downloadSpreadsheet={downloadSpreadsheet}
-              setAlert={setAlert}
+        {alert && alertContent && alertSeverity &&
+          <Alert severity={alertSeverity}>{alertContent.split('\n').map((line, index) => (
+            <React.Fragment key={index}>
+              {line}
+              <br />
+            </React.Fragment>
+          ))}</Alert>
+        }
```

Comment:
A previously done card specified that alerts should not be displayed at the top of the page, but rather underneath "Upload Program Data". The alertElement variable on line 126 was created to hold alerts and pass them downstream to UploadPage; I think you can modify that to handle the new alertContent structure, and simply remove this portion.